llvm-general/src/LLVM/General/Internal/OrcJIT.hs

```diff
@@ -138,6 +138,14 @@ findSymbol (IRCompileLayer cl dl _) symbol exportedSymbolsOnly = flip runAnyCont
     (FFI.findSymbol cl dl symbol' exportedSymbolsOnly') FFI.disposeSymbol
   decodeM symbol
 
+findSymbolIn :: IRCompileLayer -> ModuleSet -> MangledSymbol -> Bool -> IO JITSymbol
+findSymbolIn (IRCompileLayer cl dl _) (ModuleSet moduleSet) symbol exportedSymbolsOnly = flip runAnyContT return $ do
```

Comment:
`findSymbolInModule` is not quite correct, it should be `findSymbolInModuleSet`.  Personally I think sticking to the `LLVM` naming is preferable since we pretty much mirror the `C++` API anyway.